### PR TITLE
test: make test 01 and 02 use the test dracut modules

### PR DESCRIPTION
**Pull request by LaszloGombos**
make test 01 and 02 use the test dracut modules

### CODE REVIEW

File: test/TEST-02-SYSTEMD/test.sh [hunk 1]
```diff
@@ -23,59 +22,26 @@ test_run() {
...
-        find_binary plymouth > /dev/null && inst_multiple plymouth
-        cp -a /etc/ld.so.conf* "$initdir"/etc
-        ldconfig -r "$initdir"
-    )
+    "$basedir"/dracut.sh -l --keep --tmpdir "$TESTDIR" \
+        -m "test-root" \
+        -i ./test-init.sh /sbin/init \
+        -i "${basedir}/modules.d/99base/dracut-lib.sh" "/lib/dracut-lib.sh" \
+        -i "${basedir}/modules.d/99base/dracut-dev-lib.sh" "/lib/dracut-dev-lib.sh" \
+        -I "systemd-analyze" \
```
Comment (aafeijoo-suse) [inline test/TEST-02-SYSTEMD/test.sh:31]: The `test-root` module depends on `debug`, which installs `systemd-analyze`, so I think this line is not needed.

File: test/TEST-02-SYSTEMD/test.sh [hunk 2]
```diff
@@ -97,26 +62,15 @@ test_setup() {
...
-        inst_hook emergency 000 ./hard-off.sh
-    )
+    # systemd-analyze.sh calls man indirectly
+    # make the man command succeed always
     "$basedir"/dracut.sh -l -i "$TESTDIR"/overlay / \
-        -a "debug systemd" \
+        -a "test systemd" \
         -o "network kernel-network-modules" \
         -d "piix ide-gd_mod ata_piix ext3 sd_mod" \
+        -I "dd" \
```
Comment (aafeijoo-suse) [inline test/TEST-02-SYSTEMD/test.sh:71]: The `test` module depends on `debug`, which installs `dd`, so I think this line is not needed.

File: test/TEST-01-BASIC/test.sh [hunk 3]
```diff
@@ -99,20 +63,11 @@ test_setup() {
...
-        inst_hook shutdown-emergency 000 ./hard-off.sh
-        inst_hook emergency 000 ./hard-off.sh
-    )
-
     # make sure --omit-drivers does not filter out drivers using regexp to test for an earlier regression (assuming there is no one letter linux kernel module needed to run the test)
     "$basedir"/dracut.sh -l -i "$TESTDIR"/overlay / \
-        -a "debug watchdog" \
+        -a "test watchdog" \
         -d "piix ide-gd_mod ata_piix ext3 sd_mod i6300esb ib700wdt" \
+        -I "dd" \
```
Comment (aafeijoo-suse) [inline test/TEST-01-BASIC/test.sh:70]: The `test` module depends on `debug`, which installs `dd`, so I think this line is not needed.

**Review by aafeijoo-suse — COMMENTED** [review]
Also, I realized that the `test-root` module tries to install `mkdir` and `dd`, which are already installed by the `debug` module.

https://github.com/dracutdevs/dracut/blob/4a5785a8597b97957da2d8ea62e029f3da487ef8/modules.d/80test-root/module-setup.sh#L15

https://github.com/dracutdevs/dracut/blob/4a5785a8597b97957da2d8ea62e029f3da487ef8/modules.d/95debug/module-setup.sh#L18

**Comment by aafeijoo-suse** [comment]
BTW, maybe you can also remove this qemu warning :)

```
qemu-system-x86_64: -watchdog i6300esb: warning: -watchdog is deprecated; use -device instead.
```

**Review by aafeijoo-suse — APPROVED** [review]

**Review by johannbg — APPROVED** [review]

**Pull request merged**